code.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// numbers variables
+let num1 = 10;
```

Comment:
Ця і подальші змінні не перезаписується в коді, тому їх треба об'являти як константи.
```suggestion
const num1 = 10;
```
І так з наступними